ios/Classes/SwiftAirshipPlugin.swift

```diff
@@ -405,6 +405,12 @@ public class SwiftAirshipPlugin: NSObject, FlutterPlugin {
             ) as? AirshipJSON
             return result?.unWrap()
 
+        // Feature Flag
+        case "featureFlag#flag":
+            return try await AirshipProxy.shared.featureFlagManager.flag(
+                name: try call.requireStringArg()
+            )
```

Comment:
I think we need to jsonify this result. `JSONValue.wrapOpt(result).unwrap` or whatever